substrate/frame/revive/src/exec.rs

```diff
@@ -2376,9 +2383,9 @@ mod tests {
 	fn code_hash_returns_proper_values() {
 		let code_bob = MockLoader::insert(Call, |ctx, _| {
 			// ALICE is not a contract and hence they do not have a code_hash
-			assert!(ctx.ext.code_hash(&ALICE_ADDR).is_none());
+			assert!(ctx.ext.code_hash(&ALICE_ADDR).is_zero());
 			// BOB is a contract and hence it has a code_hash
-			assert!(ctx.ext.code_hash(&BOB_ADDR).is_some());
+			assert!(!ctx.ext.code_hash(&BOB_ADDR).is_zero());
```

Comment:
This should compare against an actual value. Not being zero means it can either be a non-contract account or a contract account